logevents/src/test/java/org/logevents/observers/HumioLogEventObserverTest.java

```diff
@@ -0,0 +1,208 @@
+package org.logevents.observers;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
```

Comment:
Instead of using wiremock, I'd like to see an approach more close to https://github.com/jhannes/logevents/blob/main/logevents/src/test/java/org/logevents/observers/SlackLogEventObserverTest.java which gives full test coverage to https://github.com/jhannes/logevents/blob/main/logevents/src/main/java/org/logevents/observers/AbstractHttpPostJsonLogEventObserver.java

ElasticSearchObserver predates these classes so it doesn't use the "innovations" from them

Reply:
Removed and used the approach as used in SlackLogEventObserverTest.